projects/demo/src/modules/components/data-list-wrapper/examples/import/insert-template.md

```diff
@@ -0,0 +1,6 @@
+```html
+<tui-data-list-wrapper
```

Comment:
I think we need it inside `tui-input` or `tui-select` here, because `*tuiDataList` is only relevant when used inside control.

Reply:
should I copy examples from other pages where DataListWrapper is used? e.g. https://taiga-ui.dev/components/select#wrapper